directory/src/main/java/org/ecclesia/directory/domain/Person.java

```diff
@@ -1,15 +1,18 @@
 package org.ecclesia.directory.domain;
 
+import java.io.Serializable;
 import javax.persistence.*;
```

Comment:
**Reporter**: Checkstyle
**Rule**: com.puppycrawl.tools.checkstyle.checks.imports.AvoidStarImportCheck
**Severity**: WARN
**File**: /var/jenkins_home/workspace/ecclesia_PR-6/directory/src/main/java/org/ecclesia/directory/domain/Person.java L4

Using the &#39;.*&#39; form of import should be avoided - javax.persistence.*.


*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-184856297>*